tests/TestEndpoints.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2024 Digital Bazaar, Inc.
+ *
+ * SPDX-License-Identifier: LicenseRef-w3c-3-clause-bsd-license-2008 OR LicenseRef-w3c-test-suite-license-2023
+ */
+
+import {
+  createRequestBody,
+  createVerifyRequestBody
+} from './mock.data.js';
+
+export class TestEndpoints {
```

Comment:
I am not a fan of the Endpoints class from VC 2.0 DM. I prefer the pattern used in ecdsa and eddsa test suites

Reply:
Ok I will consider this, I've been debugging a lot with vcdm 2.0 so I'm the most familiar with this.

Reply:
I will leave it as this for now since time is sensitive here